src/test/java/Homework14.java

```diff
@@ -0,0 +1,43 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+public class Homework14
+{
+
+
+
+
+    @Test
+
+        public static void navigation() {
+
+        WebDriver register = new ChromeDriver();
+        register.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://testpro.io/";
+        register.get(url);
+
+        String registrationLink = "#hel";
+        register.findElement(By.cssSelector(registrationLink)).click();
+
+        String registrationUrl = "https://bbb.testpro.io/registration.php";
+        Assert.assertEquals(register.getCurrentUrl(), registrationUrl);
+        register.quit();
+
+
+    }
+
+
+    }
```

Comment:
**How can I beautify code in IntelliJ:**

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209646734-77af1fa0-6699-442b-bffb-c245ae335987.png)